### ISSUE TITLE
F5_Recommended_Tunneling_delete_objects - flagged in every ucs

### ISSUE THREAD

**Issue by tmarfil** [f5devcentral/f5-journeys#46]
There was a change in version 3.3.1. It looks like the Journeys tool is not discerning whether the config is _using_ a tunnel, only that the default (unused?) `http-tunnel` and `socks-tunnel` config snippets exist. These config snippets exist on every BIG-IP by default.
 
```
net tunnels tunnel /Common/http-tunnel {
description "Tunnel for http-explicit profile"
profile /Common/tcp-forward
}net tunnels tunnel /Common/socks-tunnel {
description "Tunnel for socks profile"
profile /Common/tcp-forward
}
```

Everyone running Journeys will see the alert related to the socks-tunnel and http-tunnel. Maybe this is deliberate, maybe it’s not, so just be aware.

Additionally, it's not obvious how you would follow the recommendation to delete these tunnels or corresponding default profiles easily from the Configuration Utility (Web UI).

**Comment by zahajkiewicz** [comment]
Hello @tmarfil , thank you for reaching out.
We will double check the possible improvements.

**Comment by rjouhann** [comment]
Filed internal Jira SOLUTIONS-6312.

**Comment by rjouhann** [comment]
Fixed in [3.3.3](https://github.com/f5devcentral/f5-journeys/releases/tag/v3.3.3)
